### PR TITLE
Basic Proxy Example deployed to Zeit

### PR DESCRIPTION
**Pull request by sherbondy**
Hey Owen,

Not sure how we should separate out dev server experience and prod server experience.

We could have a single server entrypoint file that conditionally runs/requires dev server niceties (e.g. the webpack / live reloading) based on env var or something to that effect.

Whatever you come up with is fine by me.

Just wanted to get this proof of concept of a super basic/stripped down proxy example for one of the endpoints.

Here it is deployed to zeit via "now":
https://recipes.now.sh/

And an example of the proxied API for the recipe list endpoint:
https://recipes.now.sh/api/wtm_recipes

If you sign up for a zeit account I can add you to the team to do deployments!
It is a lot of fun.

Cheers,
Ethan

### CODE REVIEW

**Comment by sherbondy** [comment]
This is really fun:
https://recipes.now.sh/_src

:)

File: server/index.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
+const express = require('express');
+const app = express();
+
+const fetch = require('node-fetch');
+const queryString = require('query-string');
+
+// for every URL path that starts with /api/, send request to upstream API service
+function peachworksAPIURL(query_obj) {
+    let peachworks_account_id = process.env.PEACHWORKS_ACCOUNT_ID;
```
Comment (wangfowen) [inline server/index.js:9]: nitpick: `const` since it doesn't get reassigned? same with the other `let`s

File: .eslintrc [hunk 2]
```diff
@@ -4,7 +4,7 @@
     "modules": true
   },
   "env": {
-    "browser" true
+    "browser": true
```
Comment (wangfowen) [inline .eslintrc:7]: wow how did this not throw any errors before

File: server/index.js (hunk 1 above)
Comment (wangfowen) [reply server/index.js:9]: another nitpick, javascript should be camel case for variable names right?

File: server/index.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
+const express = require('express');
+const app = express();
+
+const fetch = require('node-fetch');
+const queryString = require('query-string');
+
+// for every URL path that starts with /api/, send request to upstream API service
+function peachworksAPIURL(query_obj) {
```
Comment (wangfowen) [inline server/index.js:8]: hmm? this function / comment sound like it's meant to be generic for all api requests, but then it returns specifically for wtm_recipes?

**Review by wangfowen — APPROVED** [review]
overall LGTM, will merge so I can build the dev server on top of yours. feel free to address comments separately

**Review by sherbondy — COMMENTED** [review]
Yeah all of these were oversights, did not lint prior to committing :/

Will try and setup a pre-commit hook so I don't check in garbage.

Prettier coupled with standard js, e.g., sounds like a great combination to me.

I'm going to be on vacation for the next few days but I'll try to check in & read emails every evening this week.

**Comment by wangfowen** [comment]
Oo have fun on vacation! And don't worry about checking on the project.
I'll probably also work minimally on it this week - class got switched to
afternoons, so I'm planning to just work a bit in the mornings

**Comment by wangfowen** [comment]
Hey Ethan,

Hope you had a good vacation. Tomorrow morning I'll be leaving Beijing to
go back to my grandparents' place. I'll resume working on the cards then,
and should have a ton of time to do so until the 17th.

Cheers,

**Pull request merged**